### ISSUE TITLE
NotFittedEror 

### ISSUE THREAD

**Issue by Veereshdammur** [mahesh147/Kernel-SVM#1]
After creating the SVM instance, it needs to be fitted to the training data before making predictions on the test set. Please correct me if I am wrong here.
